judge/ratings.py

```diff
@@ -28,94 +67,77 @@ def tie_ranker(iterable, key=attrgetter('points')):
         yield rank + (delta - 1) / 2.0
 
 
-def rational_approximation(t):
-    # Abramowitz and Stegun formula 26.2.23.
-    # The absolute value of the error should be less than 4.5 e-4.
-    c = [2.515517, 0.802853, 0.010328]
-    d = [1.432788, 0.189269, 0.001308]
-    numerator = (c[2] * t + c[1]) * t + c[0]
-    denominator = ((d[2] * t + d[1]) * t + d[0]) * t + 1.0
-    return t - numerator / denominator
-
-
-def normal_CDF_inverse(p):
-    assert 0.0 < p < 1
-
-    # See article above for explanation of this section.
-    if p < 0.5:
-        # F^-1(p) = - G^-1(p)
-        return -rational_approximation(math.sqrt(-2.0 * math.log(p)))
+def get_var(times_ranked, cache):
```

Comment:
Here's a trick: you can hide the cache as a default argument and it'll last for the duration of execution, i.e. do
```suggestion
def get_var(times_ranked, cache=[VAR_INIT]):
```

Reply:
Done. This improved the code readability because `solve()` no longer needs to store `cache`.